cmake/information_helpers.cmake

```diff
@@ -20,10 +20,12 @@ macro(ginkgo_git_information)
 						# NOTE: There is an error here when using the `WORKING_DIRECTORY` version.
```

Comment:
You can remove this note it seems. The syntax you are using seem to work properly (passing `Ginkgo_SOURCE_DIR` to both the command and as `WORKING_DIRECTORY`).

Reply:
Thanks for noticing, I removed that note.